diff --git a/node_modules/@minecraft/server/index.d.ts b/node_modules/@minecraft/server/index.d.ts
index daa73b5..1aa746d 100644
--- a/node_modules/@minecraft/server/index.d.ts
+++ b/node_modules/@minecraft/server/index.d.ts
@@ -11858,7 +11858,7 @@ export class System {
     /**
      * @beta
      */
-    runJob(generator: generator): number;
+    runJob(generator: Generator<void, void, void>): number;
     /**
      * @remarks
      * Runs a set of code at a future time specified by tickDelay.
